exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java

```diff
@@ -122,10 +122,10 @@
   String HTTP_SESSION_MEMORY_RESERVATION = "drill.exec.http.session.memory.reservation";
   String HTTP_SESSION_MEMORY_MAXIMUM = "drill.exec.http.session.memory.maximum";
   String HTTP_SESSION_MAX_IDLE_SECS = "drill.exec.http.session_max_idle_secs";
-  String HTTP_KEYSTORE_PATH = "javax.net.ssl.keyStore";
```

Comment:
Using the _javax_ property name for Drill property might be confusing for users, since changing that value inside Drill configuration will not reflect on the actual System property. So to avoid that confusion this property was renamed taking example from [Hadoop](https://hadoop.apache.org/docs/r2.7.2/hadoop-mapreduce-client/hadoop-mapreduce-client-core/EncryptedShuffle.html) and [IBM](https://www.ibm.com/support/knowledgecenter/en/SSZH4A_5.0.6/com.ibm.worklight.help.doc/admin/r_ssl_certificate_keystore_setup.html). Though Hadoop provides separate parameter for client/server whereas Drill has only server side config, these properties can be renamed to include _.server_ if needed. But making this name change will not affect Drill users already using the older property since backward compatibility is provided here. It will work seamlessly for them. All the below scenarios are tested. 
1) Setting javax.ssl.* properties in drill-override.conf
2) Setting javax.ssl.* properties as system options with -D option.
3) Setting ssl.* properties in drill-override.conf

So if a user provide the property values only as system property, then both Drill and JSSE will still be consuming it and work as expected.